public/animatedCounter.html

```diff
@@ -15,29 +15,44 @@ <h1>Happy Coding !</h1>
     </header>
     <section class="counters">
       <div class="container">
+
+        <div>
+            <img src="https://media3.giphy.com/media/H8DuBFems6d7UDmamS/source.gif">
+            <h2><button class="btn " onclick="start()"> start counting</button></h2>
+            </div>
+       
+        
         <div>
           <i class="fab fa-youtube fa-4x"></i>
           <div class="counter" data-target = "600000">0</div>
           <h3>Subscribers</h3>
         </div>
+        
         <div>
           <i class="fab fa-twitter fa-4x"></i>
           <div class="counter" data-target = "20000">0</div>
           <h3>Followers</h3>
         </div>
+        
         <div>
           <i class="fab fa-instagram fa-4x"></i>
           <div class="counter" data-target = "3600000">0</div>
           <h3>Likes</h3>
         </div>
+        
         <div>
           <i class="fab fa-linkedin fa-4x"></i>
           <div class="counter" data-target = "100000">0</div>
           <h3>Connections</h3>
         </div>
-      </div>
+      
+   
+    </div>
+
+    
+   
     </section>
 
-    <script src="../assets/js/animatedCounter.js"></script>
+    <script src="../assets/js/animated_Counter_(updated).js"></script>
```

Comment:
same issue here,try to make the changes in the original script file 
and also remove the updated js file here also

Reply:
here the source of the scipt is not updated to original script file..